### PR TITLE
Galerie page

### PR DESCRIPTION
**Pull request by Marie-Garde**
Début de la page Galerie

### CODE REVIEW

**Comment by TimotheeHrl** [comment]
Nickel

**Review by TimotheeHrl — DISMISSED** [review]
Nickel

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
 import "./reset.css";
 import "./App.css";
 import FrontPage from "./components/FrontPage";
-import ViewISS from './components/ViewISS';
+//import ViewISS from './components/ViewISS';
```
Comment (LoicBrassart) [inline src/App.js]: Dead code

File: src/components/GaleriePage.jsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          <Title1>
+            <FondTitre>
+              <img src="/photos/stars2.jpg" alt="stars" />
+              <H1>NASA image of the day</H1>
+              <Trait></Trait>
+            </FondTitre>
+          </Title1>
+
+          <div className="GaleriePage">
+            {" "}
```
Comment (LoicBrassart) [inline src/components/GaleriePage.jsx]: Useless line -> Remove

File: src/components/GaleriePage.jsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            //* Affichage s'il communique avec l'API, sinon il renvoie "No data
```
Comment (LoicBrassart) [inline src/components/GaleriePage.jsx]: This is *not* a comment in JSX, this text displays in your page !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You're keeping CSS files when you install Styled Components ? Please finish your transition ! You're almost there ! :muscle: 

You have some lingering indentation problems: mmaybe all the group members should install the "Prettier - Code formatter" VSC plugin ? Once configured, it autoformats on save, so you won't have any formatting problem when you commit

**Review by Guy-deMessimy — APPROVED** [review]

**Review by Guy-deMessimy — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil !

**Pull request merged**